### DISCUSSION TITLE
Mobile App

### DISCUSSION THREAD

**Discussion by Fern-Leaves** [vrcx-team/VRCX#852] [Ideas]
I personally would really like a mobile app version of VRCX so I could get a notification on my phone when my friends come online.

**Comment by slord399** [comment]
Try this
https://github.com/Nyabsi/VRCAA

**Reply by Myrkie** [reply]
vrcx doesn't have a mobile port (and will never) but a user in the community has made their own app similar to VRCX ^

**Comment by Skaveny** [comment]
> I personally would really like a mobile app version of VRCX so I could get a notification on my phone when my friends come online.

hey if you after a app for iOS/iPad/iPhone it exists

https://apps.apple.com/us/app/harmonie-for-vrchat/id6670462445

https://apps.apple.com/us/app/vrci-for-vrchat/id6746643250

**Comment by lonelymeko** [comment]
Try this,A Multi-Platform friend "monitor" for VRChat.Include Android and IOS
https://github.com/vrcm-team/VRCM
